pylot/perception/tracking/tracking_eval_operator.py

```diff
@@ -56,9 +58,10 @@ def connect(obstacle_tracking_stream, ground_obstacles_stream):
                 :py:class:`~pylot.perception.messages.ObstaclesMessage` are
                 received from the simulator.
         """
-        return []
+        finished_indicator_stream = erdos.WriteStream()
+        return [finished_indicator_stream]
 
-    def on_watermark(self, timestamp):
+    def on_watermark(self, timestamp, finished_indicator_stream):
```

Comment:
The `finished_indicator_stream` is not being used in the function. We should make sure to send watermarks both at the end of the function body and in the beginning before where the `return` is, (mistakes were made)

Reply:
We do not need to send the watermark explicitly because the operator is set to flow the watermark, which is going to be flown at the end of the watermark callback.